packages/daemon/src/rpc/methods/index.js

```diff
@@ -37,6 +37,11 @@ export default {
 
   blockchain_web3Send: blockchain.web3Send,
 
+  contacts_approveContacts: contact.approveContactsForApp,
```

Comment:
Could we make these singular rather than plural using the `contact_` prefix rather than `contacts_` for consistency with the other APIs please?